system/lib/wasmfs/backends/fetch_backend.h

```diff
@@ -6,7 +6,6 @@
 #include "wasmfs.h"
 
 extern "C" {
-
-// See library_wasmfs_fetch.js
-void _wasmfs_create_fetch_backend_js(wasmfs::backend_t);
+  // See library_wasmfs_fetch.js
+  void _wasmfs_create_fetch_backend_js(wasmfs::backend_t);
```

Comment:
Maybe revert this file?  (We don't normally indent the contents of the extern "C" block).

Reply:
Got it.